Submodels/MTSOL_call.py

```
@@ -500,18 +524,14 @@ def GetAverageValues(self,
         """
 
         # Creatge a file generator to read the output files
-        def read_file_lines(file_pattern: str):
+        def read_file_lines():
             # Generator to yield lines from files matching the pattern
-            for file in glob.iglob(file_pattern):
+            for output_file in (self.submodels_path / "MTSOL_output_files").glob("forces*"):
```

Ruff (0.8.2)

Loop control variable `output_file` not used within loop body

Rename unused `output_file` to `_output_file`

(B007)